transport/cluster_integration_test.go

```diff
@@ -74,7 +74,7 @@ func TestClusterIntegration(t *testing.T) {
 	}
 
 	// There should be at least system keyspaces present.
-	if len(c.topology.Load().(*topology).keyspaces) == 0 {
+	if len(c.topology.Load().(*Topology).keyspaces) == 0 {
```

Comment:
What's the type assertion for?

Reply:
It casts the type.

Reply:
We can always use already implemented method: 
```go
func (c *Cluster) Topology() *Topology {
	return c.topology.Load().(*Topology)
}
```